### ISSUE TITLE
The lpjgpar.json miss several table - variable combinations which have been identified

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#276]
The following list of LPJ-GUESS variables end up in the identified-missing file, while they should have become available after the ece2cmor LPJ-GUESS component merge:
`````
Amon       fco2antt      longitude latitude time                  Carbon Mass Flux into Atmosphere Due to All Anthropogenic Emissions of CO2  kg m-2 s-1   David Warlind, Tommi Bergman Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik.  Previous: Not available in the AOGCM, neither in TM5. TM5 can only provide total fluxes. Maybe sum the land use in LPJ-GUESS + fossil fuel from CEDS? This means it then has to be added in an additional post processing step. 
Amon       fco2nat       longitude latitude time                  Surface Carbon Mass Flux into the Atmosphere Due to Natural Sources         kg m-2 s-1   David Warlind, Tommi Bergman Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik.  Previous: Not available in the AOGCM, neither in TM5. TM5 can only provide total fluxes. It is not clear to us if this variable includes the contribution from land use. This variable can be calculated from the total CO2 flux provided by TM5 and the contributions from CEDS and land use from LPJ-GUESS. This "Surface Carbon Mass Flux into the Atmosphere Due to Natural Sources" could be provided by LPJ-GUESS but its definition is very vague... Should we include emissions from managed forests? Cropland? Pasture? Any managed land? If a land has a history of being managed, but now is "natural"? 
Eyr        treeFrac      longitude latitude time typetree         Tree Cover Fraction                                                         %            David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        grassFrac     longitude latitude time typenatgr        Natural Grass Area Percentage                                               %            David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        shrubFrac     longitude latitude time typeshrub        Percentage Cover by Shrub                                                   %            David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik. Included for future use 
Eyr        cropFrac      longitude latitude time typecrop         Percentage Crop Cover                                                       %            David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        baresoilFrac  longitude latitude time typebare         Bare Soil Percentage                                                        %            David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        cSoil         longitude latitude time1                 Carbon Mass in Soil Pool                                                    kg m-2       David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        cVeg          longitude latitude time1                 Carbon Mass in Vegetation                                                   kg m-2       David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        cLitter       longitude latitude time1                 Carbon Mass in Litter Pool                                                  kg m-2       David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eyr        cProduct      longitude latitude time1                 Carbon Mass in Products of Land Use Change                                  kg m-2       David Warlind                Available in LPJ-GUESS, will be cmorized by Peter Anthoni & Lars Nieradzik 
Eday       mrsll         longitude latitude sdepth time           Liquid water content of soil layer                                          kg m-2       David Warlind                Available in LPJ-GUESS, but the field will be the same as mrsol because: No frozen fraction of water 
Eday       prCrop        longitude latitude time                  Precipitation over Crop Tile                                                kg m-2 s-1   David Warlind                Available in LPJ-GUESS, but the field will be the same precipitation over crops as for the rest of the gridcell. Available in LPJ-GUESS, but the field will be the same precipitation over crops as for the rest of the gridcell. 
`````

Actually my first guess is that this is due to the changes in the `generate-lpjguess.json.sh` script which produces the  `lpjgpar.json` file.

**Comment by nierad** [comment]
Hi Thomas, 
I am sorry, that I am not more familiar with this but I don't get your point. Is that something to be fixed from the LPJG side?

**Comment by nierad** [comment]
Variables have been added to the `generate-lpjguess.json.sh` script and will now be available.

**Comment by nierad** [comment]
Ok, I have added them now, i.e. brought back the old ones...

**Comment by treerink** [comment]
Ok then please send them as a tar again

**Comment by treerink** [comment]
hi @nierad I added the list variables from this issue. For the ones which are another table variant of variables already present, I added the table name at the end of the source name in order to make them unique for LPJ-GUESS. This needs of course to be arranged in LPJ-GUESS then as well. 

From the ece2cmor3 side I have tested the current situation and now indeed zero LPJ-GUESS variables end up in the identified missing file, all arrive in the available list. So from the ece2cmor3 side this issue seems to be solved now.

Therefore closing the issue.
